apps/meteor/ee/client/voip/modal/DialPad/PadButton.tsx

```diff
@@ -20,20 +37,17 @@ const PadButton = ({
 
 	return (
 		<Button
-			m='8px'
-			pb='8px'
-			minWidth='28%'
-			display='flex'
-			flexDirection='column'
-			alignItems='center'
+			className={padButtonStyle}
 			onClick={onClick}
 			onMouseDown={onMouseDown}
 			onMouseUp={onMouseUp}
 			onTouchStart={onTouchStart}
 			onTouchEnd={onTouchEnd}
 		>
-			<Box fontSize='h2'>{firstDigit}</Box>
-			<Box fontSize='c1' color='info'>
+			<Box is='p' fontSize='h2' lineHeight='32px'>
```

Comment:
There is a real necessity of is='p'? We commonly use Box (as div) with a children text inside of it without this concern...

Reply:
@tiagoevanp When possible I like to use more meaningful tags. In this case since we are dealing with text I choose to use p. I don't think there's a problem with either element for this specific case.

Reply:
I agree with you on improve the meaningful using tags, but in this specific case, you're using inside of a button component, right? So, what about use `span` tags instead `p` tag? I think is more semantically correct once `span` is a [phrasing content](https://developer.mozilla.org/en-US/docs/Web/Guide/HTML/Content_categories#phrasing_content)